core/serial_ftdi.c

```diff
@@ -107,6 +112,9 @@ static dc_status_t serial_ftdi_sleep (void *io, unsigned int timeout)
 
 	INFO (device->context, "Sleep: value=%u", timeout);
 
+#ifdef __MINGW32__
```

Comment:
Why __MINGW32__ and not WIN32 ?

Reply:
A quick grep through our sources seems to indicate that we do indeed use WIN32 in situations like this. Would you please change that, @neolit123